### ISSUE TITLE
[Feature Request] Add link support to kanban

### ISSUE THREAD

**Issue by levindu** [FujitsuLaboratories/cattaz#17]
In kanban view, markdown link format is not recognized.
It's useful to link the list item to the page with detail.

For example:
~~~markdown
```kanban
* TODO
  * [Write English Wiki](#/page/MyWiki)
* DONE
  * Complete Feature Z
```
~~~

Also, if the `#/page/MyWiki` can be short as `MyWiki`, that'll be nice too.

**Comment by ohtake** [comment]
Thank you for a great request.
I will implement it.

> Also, if the `#/page/MyWiki` can be short as `MyWiki`, that'll be nice too.

I agree, but there is also `#/doc/index`.
To distinguish normal pages and documents, adding `#/page` is easy.

**Comment by ohtake** [comment]
Implemented in #18.
You can try it online at <http://cattaz.io/build/#/doc/app-kanban>.
If there are any issues, please reopen the issue.
